terraform/nowcasting/production/main.tf

```diff
@@ -223,7 +223,6 @@ module "pv" {
   environment             = local.environment
   public_subnet_ids       = module.networking.public_subnet_ids
   database_secret_forecast = module.database.forecast-database-secret
-  docker_version          = var.pv_version
```

Comment:
could you remove this one. In production they are pinned to a git version, so this will fail

Reply:
> could you remove this one. In production they are pinned to a git version, so this will fail

Alright, removed it in my latest commit to this PR

Reply:
sorry when I said remove, I mean could you keep this in. For this PR.

Reply:
> sorry when I said remove, I mean could you keep this in. For this PR.

basically no changes required?